default/python/AI/ResearchAI.py

```diff
@@ -849,10 +839,7 @@ def generate_classic_research_orders():
                 ):
                     res = fo.issueEnqueueTechOrder(cc_tech, insert_idx)
                     msg = "Empire is very aggressive, so attempted to fast-track %s, got result %d" % (cc_tech, res)
-                    if report_adjustments:
-                        chat_human(msg)
-                    else:
-                        debug(msg)
+                    debug(msg)
```

Comment:
You may as well remove msg and simply call:
`debug("Empire is very aggressive, so attempted to fast-track %s, got result %d" % (cc_tech, res))`

Same for all the other cases of course.

Reply:
This operation is called inline. It's a part of the refactoring menu in the IDE.